charts/enterprise-metrics/templates/graphite-querier-dep.yaml

```diff
@@ -0,0 +1,146 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-querier
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- toYaml .Values.graphite_querier.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.graphite_querier.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_querier.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        target: querier
```

Comment:
what is the label `target` for? should it be `querier`?

Reply:
In fact they shouldn't be there at all, good catch. They are for the cortex mixin which is not supporting graphite at the moment so I'm removing them from both the querier and the write proxy. a0a682e